src/main/java/io/confluent/copycat/jdbc/BulkTableQuerier.java

```diff
@@ -44,9 +44,10 @@ protected ResultSet executeQuery() throws SQLException {
 
   @Override
   public SourceRecord extractRecord() throws SQLException {
-    GenericRecord record = DataConverter.convertRecord(schema, resultSet);
+    Struct record = DataConverter.convertRecord(schema, resultSet);
     // TODO: key from primary key? partition?
-    return new SourceRecord(name, null, name, null, null, record);
+    Map<String, String> partition = new HashMap<>();
```

Comment:
Not critical, but for simple maps like this it can be nice to use `Collections.singletonMap` to keep it concise.